pkg/imgutil/snapshotter_test.go

```diff
@@ -175,3 +185,13 @@ func checkStargzSnapshotterAnnotataions(t *testing.T, a map[string]string) {
 	_, ok := a["containerd.io/snapshot/remote/urls"]
 	assert.Equal(t, ok, true)
 }
+
+func checkSociSnapshotterAnnotataions(t *testing.T, a map[string]string) {
+	assert.Check(t, a != nil)
+	_, ok := a["containerd.io/snapshot/remote/soci.size"]
+	assert.Equal(t, ok, true)
+	_, ok = a["containerd.io/snapshot/remote/image.layers.size"]
```

Comment:
Is the image.layers.size specified to soci? Where do you get this information? Is there a link available to put in the comment?

Reply:
I got these from the values used to set annotations in the soci source. 
https://github.com/awslabs/soci-snapshotter/blob/ffd306b79f413cb2cd6ebfc05cadfb13cf4d325d/fs/source/source.go#L82C2-L82C61
https://github.com/awslabs/soci-snapshotter/blob/ffd306b79f413cb2cd6ebfc05cadfb13cf4d325d/fs/source/source.go#L185

Reply:
I think you can put the link above the function and explain they are from here https://github.com/awslabs/soci-snapshotter/blob/ffd306b79f413cb2cd6ebfc05cadfb13cf4d325d/fs/source/source.go#L80